### ISSUE TITLE
💄Is there a way to disable the color fill on links for an image gallery item?

### ISSUE THREAD

**Issue by GrantsTracks2121** [nunocoracao/blowfish#254]
I'm evaluating the Blowfish theme and there's a lot to like about it! I keep finding cool little features that I think I'll use in the future. Nice job on the theme.

**Describe the bug**
I'm posting this under "Bug reports" but I'm not sure it's a bug; it's more like two pieces of software being put together and not playing completely nice together.

Displaying image galleries is important to me, so I added "Hugo Easy Gallery"  to the theme. The gallery works fine, but When one hovers over a "Hugo Easy Gallery" item, the item fills with color. This masks out the image behind it and also caption text that appears on hover.

When I change the theme's color scheme, that fill color changes. So I conclude the problem is with links being filled on hover". I honestly have spent a few hours trying to hunt down the source of this behavior in the css, but I just don't have the knowledge or experience to figure out a solution.

**To Reproduce**
I doubt you'll need to or want to do all this, 😀 but here it goes:
1. Install Hugo Easy Gallery according to instructions here.
2. Make sure images are stored in "/static/images/Name-of-post/" in the Hugo site
3. Call the gallery with the built in shortcode e.g. {{< gallery dir="/img/arduino/" />}}
4. Mouse over a gallery item in the rendered Hugo site.

**Expected behavior**
A color would not fill the image gallery item.

**Screenshots**
Here's a animated gif to show you want I mean:
[The Fill on Hover effect on an image gallery item](https://grantstracks.com/wp-content/uploads/The_Evil_Fill_on_Hover_Effect.gif)

Not sure if it helps at all but here's the section of HTML that Hugo Easy Gallery renders:
[The Hugo Easy Gallery HTML code](https://grantstracks.com/wp-content/uploads/TheGallerycode.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 107.0.5304.110

**Hugo & Blowfish versions**
Hugo version: hugo v0.105.0+extended darwin/amd64
Blowfish version: don't know how to determine - downloaded from Github Nov. 15, 2022

**Comment by nunocoracao** [comment]
Hello @GrantsTracks2121, as you are integrating a new component into the theme I don't think this should be a bug. As it's something that the theme shouldn't support out of the box, especially given that project is not maintained since feb. Having said this I do want to try to help. It would be easier if I can look at the code though, do you have a repo link that I can dig?

At this stage it seems that either 1) we can improve the theme to not do that, or 2) I can try to check how a CSS override could make it work. 

Let me know if this works for you.

**Comment by GrantsTracks2121** [comment]
Hi Nuno,

I might need to apologize for not answering sooner - here in Github at least. I simply replied to your very timely message directly from my mail client and since I don't see that reply in the thread here, I'm not at all sure that you received it. I'm not entirely new to Github, but I am completely new to using the site to communicate. 
But more importantly...I was able to solve the problem - with some css duct tape.

In the css that controls the Hugo Easy Gallery, I added a "pointer-events: none;" to the <a> tag inside the "img" class that is applied to each item in the gallery. That disabled the color fill behaviour but there's still a clickable link inside the item.

Since the pointer events were "turned off" the mouse cursor displayed then as an arrow over the link so I added a "cursor: pointer;" to the "img" class to fake a pointer to indicate a there's a clickable link. Like I say, duct tape. But it works!

Thanks for your time and thanks for your work on the theme. It's full speed ahead now transitioning from WordPress to a Hugo-based blog.

Take care!
